entrypoint.sh

```diff
@@ -159,11 +161,27 @@ if [[ -n "${MUMBLE_SUPERUSER_PASSWORD}" ]]; then
 fi
 
 # Show /data permissions, in case the user needs to match the mount point access
-echo "Running Mumble server as uid=$(id -u) gid=$(id -g)"
+echo "Entrypoint running as uid=$(id -u) gid=$(id -g)"
+echo "Preparing Mumble server to run as uid=$(MUMBLE_UID) gid=$(MUMBLE_GID)"
+
+if [[ "$MUMBLE_UID" != "0" ]]; then
+  groupmod -og "$MUMBLE_GID" mumble
+  usermod -ou "$MUMBLE_UID" mumble
+  if [[ "$MUMBLE_NO_CHOWN" = true ]]; then
```

Comment:
Seems like a wrong if condition

Reply:
Code not tested yet (draft) during my travel.
If uid = 0 -> do nothing (changing group to root isn't necessary and possible)
if no_chown -> don't to the chown step

Reply:
Indeed, double negation is always bad. I've reverse the logic.